.github/workflows/constants.yaml

```diff
@@ -0,0 +1,65 @@
+name: Set constants
+
+on:
+  workflow_call:
+    inputs:
+      event_name:
+        required: true
+        type: string
```

Comment:
Is this input needed / being used anywhere?

Reply:
@rashidnhm I thought there should be always some inputs with the outputs context :laughing: Fixed it!